activesupport/lib/active_support/json/coder.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module ActiveSupport
+  module JSON
+    class Coder
+      def initialize(options = nil)
+        @options = options || {}
+      end
+
+      def dump(obj)
+        JSON.encode(obj)
+      end
+
+      def load(json)
+        JSON.decode(json, @options) unless json.blank?
```

Comment:
As mentioned above, I get why the `.blank?` check, but this is an Active Record concern leaking into ActiveSupport. We should figure out a way not to have this.